cfgov/housing_counselor/templates/housing_counselor/index.html

```diff
@@ -178,7 +178,7 @@ <h1>Find a housing counselor</h1>
                                                     <div class="hud_hca_api_num">{{ forloop.counter }}.</div>
                                                     <div class="hud_hca_api_counselor_info_container">
                                                         <div class="hud_hca_api_counselor">
-                                                            {% if valid_url %}<a href="{{ signed_url }}" class="icon-link icon-link__external-link">{% endif %}
+                                                            {% if valid_url %}<a href="????" class="icon-link icon-link__external-link">{% endif %}
```

Comment:
Set this to `#` and add `TODO` in a comment above it so we can find it in a search.

Reply:
Seems like we can leave the existing page working as it is, right?